internal/db/queryBuilder.go

```diff
@@ -136,25 +148,38 @@ func (qb *queryBuilder) queryWithRemovals() string {
 	%s
 	)
 
-	SELECT path, mode, size, bytes, packed, deleted
+	SELECT path, mode, size, bytes, packed, deleted, hash
 	FROM updated_objects
-	UNION
-	SELECT path, mode, size, bytes, packed, deleted
+	UNION ALL
```

Comment:
I changed this since `deleted` is always different between `updated_objects` and `removed_objects`